securedrop/journalist_templates/edit_account.html

```diff
@@ -61,8 +67,14 @@ <h2>{{ gettext('Reset Password') }}</h2>
 
 <form action="{{ password_reset_url }}" method="post" id="new-password" class="login-form">
    {% if not user or g.user == user %}
-  <p><input type="password" name="current_password" placeholder="{{ gettext('Current Password') }}"></p>
-  <p><input name="token" id="token" type="text" placeholder="{{ gettext('Two-factor Code') }}"></p>
+  <div>
+    <label for="current_password" class="visually-hidden">{{ gettext('Current Password') }}</label>
```

Comment:
Not sure if it's in scope, but these labels seem like they should be shown for all users instead of relying solely on the placeholder text.

Reply:
I have mixed feelings about the label/placeholder usage across SecureDrop forms too.  In the Journalist Interface:

* labels are visible everywhere fields require explication (e.g., `admin_add_user.html`);
* labels are visible and augmented with placeholders where fields have logical relations (e.g., label "Is using a Yubikey" + placeholder "HOTP Secret"); and
* labels are visually hidden in favor of placeholders, but still present in the accessibility tree, where fields have logical relations familiar enough to be left implicit (e.g., `login.html`'s three fields and the two fields you've cited in `edit_account.html`).

In this case in `edit_account.html`, it seems to me that the two fields here:

1. visually imitate the three fields of the login form;
2. have explanatory text nearby in— <https://github.com/freedomofpress/securedrop/blob/cbfc08f73c56afa970fbdadf8ecbffe4d6d2e7e3/securedrop/journalist_templates/edit_account.html#L65>—and
3. still benefit from their placeholder text.

So, for kicks, here is c9eb580a74e9b066d71724a5845c37d1aa6342d7, where these fields are grouped in a `fieldset` with an explanatory `legend`, otherwise keeping the `login.html`-like field layout.  I'll be curious to know whether you and @zenmonkeykstop like this or think it's more trouble than it's worth.  :-)

Reply:
I like the fieldset, it's a minimal change but IMO improves the usability of it.